backend/app/api/agents.py

```
@@ -380,7 +408,9 @@ async def register_agent(
             raise ValueError(f"Failed to register agent {request.agent_id}")
 
     except Exception as e:
-        raise handle_service_error("agent_registration", e, user_info.get("user_id"))
+        raise handle_service_error(
+            "agent_registration", e, user_info.get("user_id")
+        )
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)